droidlet/perception/craftassist/voxel_models/instance_segmentation/mask_output_viewer/index.html

```diff
@@ -0,0 +1,93 @@
+<!-- Voxel model output viewer -->	
+<!-- Bootstrap v3.0.3 -->	
+
+<!-- Copyright (c) Facebook, Inc. and its affiliates.
+   This source code is licensed under the MIT license found in the
+   LICENSE file in the root directory of this source tree. -->
+<html>
+    <head>
+        <link	
+        rel="stylesheet"	
+        href="https://stackpath.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"	
+        />	
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>	
+        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>	
+        <script>
+            sessionStorage.setItem('gtScene', window.prompt("URL to Ground Truth Scene",""));
+            sessionStorage.setItem('modelOutput', window.prompt("URL to Model Output Scene",""));
+        </script>
+    </head>
+
+    <body>
+        <section class="container"	id="heading">	
+        <div class="row col-xs-12 col-md-12">
+            <div class="panel panel-primary">		
+            <div class="panel-heading"></div>
+            <div class="collapse in" id="instructionsWrapper">	
```

Comment:
Is is possible for the user to hide the instructions and get more room? 

I see you're using "collapse" but I didn't see the toggle in the image

Reply:
Good catch; I took out the collapse functionality because the instructions are very short and everything fits in one window with no scroll, but I didn't update all of the classes.  I'll clean it up a bit.